src/main/java/racingcar/domain/RacingCarGame.java

```diff
@@ -28,24 +26,30 @@ private List<Car> createCars(List<String> carNames) {
         return carList;
```

Comment:
자동차 목록을 생성할때 스트림을 활용해서 이렇게도 쓸 수 있을거 같아요.
```suggestion
        return carNames.stream()
                .map(Car::new)
                .collect(Collectors.toList());
```